protocols/gossipsub/src/handler.rs

```diff
@@ -475,14 +431,17 @@ impl ConnectionHandler for Handler {
                                         Some(OutboundSubstreamState::WaitingOutput(substream));
                                 }
                                 Err(e) => {
-                                    error!("Error sending message: {}", e);
-                                    return Poll::Ready(ConnectionHandlerEvent::Close(e));
+                                    log::debug!(
+                                        "Outbound substream error while sending output: {e}"
+                                    );
+                                    self.outbound_substream = None;
+                                    break;
                                 }
                             }
                         }
                         Poll::Ready(Err(e)) => {
-                            error!("Outbound substream error while sending output: {:?}", e);
-                            return Poll::Ready(ConnectionHandlerEvent::Close(e));
+                            error!("Outbound substream error while sending output: {e:?}");
```

Comment:
Minor consistency alignment.

```suggestion
                            log::debug!("Outbound substream error while sending output: {e}");
```